setup.cfg

```diff
@@ -24,9 +22,7 @@ classifier =
 [options]
 install_requires =
     six>=1.9.0
-    futures>=3.0;python_version=='2.7'
-    monotonic>=0.6;python_version=='2.7'
-    typing>=3.7.4.1;python_version=='2.7'
+python_requires = >=3.6, <4
```

Comment:
you sure it does not work on Python 4?

Reply:
Officially announced, that changes like 2 -> 3 is not expected in the future. If it will be changed - it will be reason to make such limitation

Reply:
That's what some popular projects do ([urllib3](https://github.com/urllib3/urllib3/blob/55465173b3eb233e14159908834c64367c28e203/setup.py#L109)) and what some package managers force you to do ([poetry](https://github.com/python-poetry/poetry/issues/1413)). I explored this topic a little and found that [people in PyPA prefer to not have an upper bound](https://discuss.python.org/t/use-of-less-than-next-major-version-e-g-4-in-python-requires-setup-py/1066/3). So, let it be this way. Removed `<4`.